### DISCUSSION TITLE
Fill Select multiple

### DISCUSSION THREAD

**Discussion by true-perfect-code** [ant-design-blazor/ant-design-blazor#1769] [General]
Hello  

I have a SELECT list (multiple is set) and I want to fill it with pre-selected values before displaying.

Problem is that the binding variable is of type IEnumerable<string> and I can't just use an ADD.

Does anyone have an idea?

Thanks
perfect-code

**Comment by ElderJames** [comment]
How about the DefaultValues?

**Reply by true-perfect-code** [reply]
Thank you ElderJames for the suggestion. Of course I tried it before I posted here, didn't work. It didn't work with simple value like DefaultVaule="@myValue" and then string myValue = "Test-Item".

Actually I need Tag DefaultValues, but unfortunately there is no example that shows how to set DefaultValues with an IEnumerable variable!

Could someone create a small example in the doc for setting DefaultValue and DefaultValues?

Thanks

**Reply by ElderJames** [reply]
Could you please offer a demo for your scenes?
